code/postgres/src/main/scala/postgres/clients/BulkSenderPostgres.scala

```diff
@@ -16,30 +16,29 @@ import com.github.blemale.scaffeine.Cache
 import com.snowplowanalytics.stream.loader.EmitterJsonInput
```

Comment:
I think it is probably cleaner to have a BulkSenderTimescaledb.scala as an alternative to changing the Postgres one?